app/controllers/trains_controller.rb

```diff
@@ -3,10 +3,24 @@ def index
     @trains = TrainApi.new.index
   end
 
+  def edit
+    flash["errors"] = nil
```

Comment:
no need for this

Reply:
> no need for this

I added this else when i try to click edit, the previous errors from the last update request will show on the edit page. If that makes sense.
because the `flash["errors"]` isnt assigned to nil so it will show up when i click edit train again
Should i still remove it?

Reply:
they should not remain on page refresh, there shouldnt be a need for it i think.
the flash message should be getting cleared on refresh

Reply:
> the flash message should be getting cleared on refresh

the flash message still persists on refresh tho

so when i click edit and input invalid params then click submit, the error messages show up
next, i click the train header button and click edit on any train in the index and the edit page shows for that train but with also the error messages from the previous invalid params

Reply:
okay leave it in then, i think theres a way to set it so that the error doesnt persist but i guess its more then just using `flash[]`